doc/ref_arch/openstack/chapters/chapter05.md

```diff
@@ -108,10 +125,10 @@ This section illustrates some of the Interfaces provided by OpenStack; the exhau
 | OpenStack Client (python) | https://docs.openstack.org/python-openstackclient/pike/index.html  | Version 3.2.1 |
 
 ### 5.3.2. Kubernetes Interfaces
-The Kubernetes APIs are available at https://kubernetes.io/docs/concepts/overview/kubernetes-api/. 
+The Kubernetes APIs are available at https://kubernetes.io/docs/concepts/overview/kubernetes-api/.
```

Comment:
__query__ - Why kubernetes API interfaces needs to be mentioned for RA1 Interface and APIs?

Reply:
@deebhatia RA-1 supports hosting of OpneStack services on both virtual and physical machines (bare metal) -- not containerised workloads just containerised OpenStack services; containerised OpenStack services is now quite popular with support from multiple OpenStack vendors.

Reply:
That I understand. However, document doesn't mention it at all. We should mention that explicitly. 
Another thing is that are we assuming here that the containerised services will only be Kubernetes managed?

Reply:
@deebhatia [Chapter 4.3.2.](https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/openstack/chapters/chapter04.md) explicitly states, "The services can be containerized or VM hosted as long as they provide the high availability principles described above."

Currently, as far as I know, kubernetes has been chosen for container management.

Reply:
Understood. Thanks!